samples/rqd/blender/Dockerfile

```diff
@@ -30,5 +30,8 @@ RUN tar -xvf blender.tar.xz \
 
 RUN rm blender.tar.xz
 
+# Add Blender path as environment variable
```

Comment:
Let's split this file's changes into its own PR since it is unrelated to the pynput change.

+1 to Diego's comment, please confirm whether the fallback behavior is working without this change. RQD will produce the pynput warning but it should still work.

Reply:
> Let's split this file's changes into its own PR since it is unrelated to the pynput change.

Noted. Commit removed and added in PR https://github.com/AcademySoftwareFoundation/OpenCue/pull/1319

> +1 to Diego's comment, please confirm whether the fallback behavior is working without this change. RQD will produce the pynput warning but it should still work.

Checked. 
Despite the error, the functionality remains unaffected as shown in the screenshot attached.
![Selection_104](https://github.com/AcademySoftwareFoundation/OpenCue/assets/24616975/c83c0192-77f6-48bb-bd35-0e501980fc82)

To check if there's an alternative, I also looked around to see if there was a way to detect user activity on the host machine _from within_ a running Docker container but couldn't find anything useful.